scripts/wait_for_listening_port.py

```diff
@@ -29,7 +29,7 @@ def _wait_for_port(port, server_pid, timeout) -> Tuple[bool, str]:
         if not _pid_exists(server_pid):
             return False, f"Server PID {server_pid} is not running."
         try:
-            socket.create_connection(("localhost", port), timeout=0.5)
+            socket.create_connection(("localhost", port), timeout=1.5)
```

Comment:
Loosen this a bit.... the purpose of this test is not to ensure low latency.  Who knows how slow Windows / WSL can get.